package.json

```diff
@@ -79,7 +79,7 @@
     "jest-junit": "^16.0.0",
     "jest-localstorage-mock": "^2.4.26",
     "jest-watch-typeahead": "^2.2.2",
-    "msw": "^2.0.13",
+    "msw": "^1.2.2",
```

Comment:
I want to downgrade to a version where I can properly mock the server, since I have not been able to make it work with the latest one. This only affects the tests, not the actual app